packages/framework-provider-kubernetes/src/library/events-adapter.ts

```diff
@@ -29,7 +29,7 @@ export const store = async (
 const isNewerThan = (isoString: string) => (key: string) => {
   const keyIsoString = key.split('_')[5]
   if (!keyIsoString) throw new Error(`No ISO string in key ${key}`)
-  return new Date(keyIsoString) >= new Date(isoString)
+  return new Date(keyIsoString) > new Date(isoString)
```

Comment:
Here the operator is isNewerThan so equal dates should not be included. Actually there was a bug when you reduced 5 events one of them where counted twice, so having a balance of 1000 and adding 5 events of increments of 10 were resulting in 1060 in place of 1050